bHY6yh-de1319/cod-vyg67d_bHY6yh-de1319.xml

```diff
@@ -0,0 +1,327 @@
+           patientibus. </quote> 
+        </p>
+           <p xml:id="bHY6yh-de1319-d1e302">
+           <lb ed="#O"/> ¶ <ref xml:id="bHY6yh-de1319-Rd1e351">Salomon in pro<!--line bumps up-->uerbiis. capitulo 5. <!--line bumps back down--> </ref> <quote xml:id="bHY6yh-de1319-Qd1e306"> Respicit dominus 
+           <lb ed="#O"/>vias hominis: et omnes gressus illius
+           <lb ed="#O"/>considerat. </quote>
+        </p>
+           <p xml:id="bHY6yh-de1319-d1e315"> 
+           ¶ <ref xml:id="bHY6yh-de1319-Rd1e363">11. capitulo</ref> <quote xml:id="bHY6yh-de1319-Qd1e317">non proderunt di<lb ed="#O" break="no"/>uitie 
+           in die ultionis. Iustitia autem 
+           <lb ed="#O"/>liberabit amorte. </quote>
+        </p>
+           <p xml:id="bHY6yh-de1319-d1e323"> 
+           ¶ <ref xml:id="bHY6yh-de1319-Rd1e373">15. capitulo</ref> <quote xml:id="bHY6yh-de1319-Qd1e325">In omniloco 
+           <lb ed="#O"/>oculi domini contemplantur bonos et
+           <lb ed="#O"/>malos. </quote>
+        </p>
+           <p xml:id="bHY6yh-de1319-d1e333">
+           ¶ <quote xml:id="bHY6yh-de1319-Qd1e335">Infernus et perditio coram 
+           <lb ed="#O"/>domino:quanto magis corda filiorum 
+           <lb ed="#O"/>hominum. </quote>
+        </p>
+           <p xml:id="bHY6yh-de1319-d1e341"> 
+           ¶ <ref xml:id="bHY6yh-de1319-Rd1e394">16. capitulo</ref> <quote xml:id="bHY6yh-de1319-Qd1e343">Omnes viae hominum 
+           <lb ed="#O"/>patent oculis domini. </quote>
+        </p> 
+           <p xml:id="bHY6yh-de1319-d1e349">
+           ¶ <ref xml:id="bHY6yh-de1319-Rd1e408">22. capitulo</ref> <quote xml:id="bHY6yh-de1319-Qd1e351">non facies 
+           <lb ed="#O"/>violentiam hominum pauperi quia pauper
+           <cb ed="#)" n="b"/>
+           <lb ed="#O"/>est: neque conteres egenum in porta 
+           <lb ed="#O"/>quia dominus iudicabit causam eius. </quote> 
+        </p> 
+           <p xml:id="bHY6yh-de1319-d1e361">
+           ¶ <ref xml:id="bHY6yh-de1319-Rd1e424">29. capitulo</ref> <quote xml:id="bHY6yh-de1319-Qd1e363">a domino iudicium egreditur. 
+           <lb ed="#O"/>singulorum. </quote>
+        </p>
+           <p xml:id="bHY6yh-de1319-d1e370">
+           ¶ <ref xml:id="bHY6yh-de1319-Rd1e434">Idem in eccl. capitulo 3.</ref> <quote xml:id="bHY6yh-de1319-Qd1e372">iu<lb ed="#O" break="no"/>stum et impium iudicabit dominus. </quote>
+        </p>
+           <p xml:id="bHY6yh-de1319-d1e378"> 
+           ¶ <ref xml:id="bHY6yh-de1319-Rd1e444">12. capitulo et fine</ref> <quote xml:id="bHY6yh-de1319-Qd1e380">cuncta que fiunt adducet dominus 
+           <lb ed="#O"/>in iudicium pro omni ertato: siue bo<lb ed="#O" break="no"/>num 
+           siue malum sit. </quote> 
+        </p>
+           <p>
+           <!--section transcriped in other vatican text not found in this one-->
+         </p>
```

Comment:
You're on the right track here with this omitted paragraph. But it's very important not to lose the "paragraph id". When the content of a paragraph is missing, I'd also like you to right "[omitted]" so that it can be easily found. In the correction below, I've added the Id back and then added the word [omitted]. If you can make this change, then I'll accept the pull request :)

```suggestion
           <p xml:id="bHY6yh-de1319-d1e388">
             [omitted]
             <!--section transcriped in other vatican text not found in this one-->
            </p>
```